src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/FluentBindingProviderExtensions.cs

```diff
@@ -11,16 +11,14 @@ namespace Microsoft.Azure.WebJobs.Host
 {    
     // Internal Extension methods for setting backwards compatibility hooks on certain bindings. 
     // This keeps the hooks out of the public surface. 
-    internal static class FluidBindingProviderExtensions
+    internal static class FluentBindingProviderExtensions
```

Comment:
Yes. Bot PR is: [https://github.com/Microsoft/BotBuilder-WebJobs-BotExtension/pull/6](https://github.com/Microsoft/BotBuilder-WebJobs-BotExtension/pull/6)

---
In reply to: [117080116](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r117080116) [](ancestors = 117080116)